### PR TITLE
Added remaining parts for unsigned resources

### PR DESCRIPTION
**Pull request by superwofy**
There were a few missing labels. Final result:

image

P.S. What do you think about using a toggle in the advanced settings instead? I suppose that would make it easier for other users since they wouldn't have to recompile the app. However, the difficulty of patching the library may render that irrelevant anyway.

### CODE REVIEW

File: app/src/test/java/me/hufman/androidautoidrive/notifications/StatusbarAppTest.kt [hunk 1]
```diff
@@ -91,7 +91,7 @@ class StatusbarAppTest {
 	fun testNotificationCenterController() {
 		val mockServer = MockBMWRemotingServer()
 		IDriveConnection.mockRemotingServer = mockServer
-		val app = ID5StatusbarApp(iDriveConnectionStatus, securityAccess, carAppResources, graphicsHelpers)
+		val app = ID5StatusbarApp(iDriveConnectionStatus, securityAccess, carAppResources, mock(), graphicsHelpers)
 		val controller = app.statusbarController
 		val event = app.notificationEvent
```
Comment (hufman) [inline app/src/test/java/me/hufman/androidautoidrive/notifications/StatusbarAppTest.kt:97]: I think there's one more in testID5Popup that the [unit tests](https://github.com/BimmerGestalt/AAIdrive/actions/runs/10207416351/job/28242130977?pr=808) are complaining about:
```
file:///home/runner/work/AAIdrive/AAIdrive/app/src/test/java/me/hufman/androidautoidrive/notifications/StatusbarAppTest.kt:146:86 Type mismatch: inferred type is GraphicsHelpers but CarAppWidgetAssetResources was expected
```

**Comment by hufman** [comment]
Cool! Up to you if you want to have the BMW One icon labelled, I believe the app performs absolutely no UI functionality currently :)
I don't see a need to add an extra option currently. If we get more users of this feature, we can easily toggle it on in the default build: one extra roundtrip in two separate threads won't hurt much, especially since texts.zip is so small, and the try/catch makes it harmless for everyone else.
Just gotta fix up one unit test and I'll merge, and then merge (probably squash with your name as the sole author) this branch to the main branch :D

**Comment by superwofy** [comment]
> Cool! Up to you if you want to have the BMW One icon labelled, I believe the app performs absolutely no UI functionality currently :) I don't see a need to add an extra option currently. If we get more users of this feature, we can easily toggle it on in the default build: one extra roundtrip in two separate threads won't hurt much, especially since texts.zip is so small, and the try/catch makes it harmless for everyone else. Just gotta fix up one unit test and I'll merge, and then merge (probably squash with your name as the sole author) this branch to the main branch :D

You're right, it does nothing. When you click into it it prints a message to the tune of "This app is used to push notifications...". It did upset me that it had no label though.  

On that note, I wonder if there's anything we can do with the widgets. If you had a look at the fs dump you might have seen files with the ".pbw" extension. These are binary compiled widgets for the hmi and are used for various things like displaying images/animations in the sidebar. I experimented with changing them before with mixed results. What's interesting is that with the app, they're provided as un-compiled text (ptw). What purpose do they serve? Can they be used for something interesting?

**Comment by hufman** [comment]
I haven't explored the widget system very much... I think the cert doesn't validate them, which is exciting, but the signed ui_description has to reference them in the first place. The only place where AAIdrive uses them is the ID5 Notification Popup. It looked like it had some intriguing options for defining the layout and properties of the elements, but I haven't experimented at all. I'd be excited to hear what you find :)

**Pull request merged**